### PR TITLE
Npm command alternative

### PR DESCRIPTION
**Pull request by rosexw**
Hi Georg,

I have added an alternative command to running the web server using npm. I added this sentence to the bottom of the section: ### Working on the dashboard.

Rose

### CODE REVIEW

File: probe-dictionary/explorer.css [hunk 1]
```diff
@@ -8,6 +8,18 @@ body {
     margin-right: 2px;
 }
 
+/* Fix the NavBar to the top of the page*/
+.fixed-navbar {
```
Comment (georgf) [inline probe-dictionary/explorer.css]: It looks like some commits for #448 into this PR :)

**Comment by georgf** [comment]
Chris, you probably have a more informed opinion on the README addition.

File: README.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ A local webserver is really helpful for running a version of the site on the sam
 
 Permalink buttons (in the top right hand corner of the main dashboards) will not work when running the site on local servers. This is because they are shortened with bit.ly, which doesn't allow local links.
 
+Alternatively, we can run on [npm web server](https://www.npmjs.com/package/http-server) using: `npm install http-server -g`
```
Comment (chutten) [inline README.md]: I would prefer this moved up closer to the instructions for the python web server, and presented in a similar way.

**Review by chutten — CHANGES_REQUESTED** [review]

**Comment by rosexw** [comment]
OK sure, I will edit this and then submit another pull request for review. 

Thank you for your feedback, Georg and Chris!

**Comment by chutten** [comment]
You still need to remove the other unrelated changes Georg mentioned. And you get to remove the original npm mention.

Have you ever used `git rebase -i`? It might be very useful to you right now, allowing you to remove commits from your local repo that you don't want any more. You might want to `git checkout -b npm-instructions` to switch to a new branch first, in case you want to split your changes.

**Comment by rosexw** [comment]
Removed the unrelated commits with `git rebase -i` and the outdated information.

Thanks, @chutten!

**Review by chutten — APPROVED** [review]

**Comment by chutten** [comment]
Thank you for your contribution!

**Pull request merged**